config-windows.md

```diff
@@ -94,3 +94,23 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsHyperV" />HyperV
+
+`hyperv` is an OPTIONAL field of the Windows configuration. If present, the container will be run with Hyper-V isolation. If omitted, the container will be run as a Windows Server container.
```

Comment:
Ok, if that feels too strong, maybe relaxing the second half to a `SHOULD` would be appropriate?  ie, if Hyper-V is requested via config, the container either MUST run via Hyper-V, or there must be an error, but otherwise, the behavior can be determined by the runtime if necessary?